server/config/config.go

```diff
@@ -739,7 +739,7 @@ type ScheduleConfig struct {
 	HotRegionsWriteInterval typeutil.Duration `toml:"hot-regions-write-interval" json:"hot-regions-write-interval"`
```

Comment:
It's kind of misleading, can we find a better name?

Reply:
how about change to hot region record interval?

Reply:
But seem there is a compatibility problem since this config is involved in v5.3.0.

Reply:
let's just keep it for now.